docs/guides/kds-sync-name.md

```diff
@@ -0,0 +1,186 @@
+# What is the name of the resource after KDS sync?
+
+## Old policies (i.e. TrafficRoute, TrafficPermission)
+
+* CRD scope: Cluster
+* Kuma scope: Mesh
+
+### Global(k8s/universal) -> Zone(k8s/universal)
+
+```
+nameInZone := "${truncate(nameInGlobal, 236)}-${hash(mesh, nameInGlobal)}"
+```
+
+Example: `my-policy -> my-policy-fvdw2c2wx44fv4xz`
+
+### Zone(k8s/universal) -> Global(k8s/universal)
+
+Not supported
```

Comment:
Are we sure of how it'll work once we relax this limitation?

Reply:
Ha actually I think we allow zone side writes only for new policies right?

Reply:
Yes, only new ones